payment_api/controller/payment_methods/handler.go

```diff
@@ -0,0 +1,241 @@
+package paymentmethods
+
+import (
+	"encoding/json" // "encoding/json" // package to encode and decode the json into struct and vice versa
+
+	"github.com/gorilla/mux" // used to get the params from the route
+	"log"
+	"net/http" // used to access the request and response object of the api
+
+	"github.com/jpgsaraceni/mvp-backend/payment_api/models"
+	pm_service "github.com/jpgsaraceni/mvp-backend/payment_api/services/payment_methods"
+)
```

Comment:
É padrão do Go colocar primeiro os imports da standard lib, seguido por pacotes de terceiros, seguido pelos pacotes do próprio repositório e.g.:

```suggestion
import (
	"encoding/json" // package to encode and decode the json into struct and vice versa
	"log"
	"net/http" // used to access the request and response object of the api
	
	"github.com/gorilla/mux" // used to get the params from the route

	"github.com/jpgsaraceni/mvp-backend/payment_api/models"
	pm_service "github.com/jpgsaraceni/mvp-backend/payment_api/services/payment_methods"
)
```